File: PYTHON/Program/CPU.py
from Komponen import Komponen

class CPU(Komponen):
    def __init__(self, jumlahCore=0, kecepatanGhz=0.0, merk="", nama=""):
        super().__init__(merk, nama)
        self.jumlahCore = jumlahCore
        self.kecepatanGhz = kecepatanGhz

    def setJumlahCore(self, jumlahCore):
        self.jumlahCore = jumlahCore

    def setKecepatanGhz(self, kecepatanGhz):
        self.kecepatanGhz = kecepatanGhz

    def getJumlahCore(self):
        return self.jumlahCore

    def getKecepatanGhz(self):
        return self.kecepatanGhz

File: JAVA/Program/CPU.java
package Program;
public class CPU extends Komponen {
    private int jumlahCore;
    private float kecepatanGHz;

    public CPU() {
    }

    public CPU(int jumlahCore, float kecepatanGHz, String merk, String nama) {
        super(merk, nama);
        this.jumlahCore = jumlahCore;
        this.kecepatanGHz = kecepatanGHz;
    }

    public void setJumlahCore(int jumlahCore) {
        this.jumlahCore = jumlahCore;
    }

    public void setKecepatanGHz(float kecepatanGHz) {
        this.kecepatanGHz = kecepatanGHz;
    }

    public int getJumlahCore() {
        return jumlahCore;
    }

    public float getKecepatanGHz() {
        return kecepatanGHz;
    }
}
